src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  let selectedTab;
+
+  if (tabs.some(tab => tab.id === selectedTabId)) {
+    selectedTab = tabs.find(tab => tab.id === selectedTabId);
+  } else {
+    [selectedTab] = tabs; // taking first element of an array
+  }
```

Comment:
try to rewrite without let. 
In global component scope let is not a good idea because we get worse readability and code support.